### PR TITLE
Added skmt/15k keyboard

### PR DESCRIPTION
**Pull request by satorusaka**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/skmt/15k/15k.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 Jimmy Lye (@satorusaka)
```
Comment (zvecr) [inline keyboards/skmt/15k/15k.h:1]: This file should be removed

File: keyboards/skmt/15k/skmt_15k_macropad.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (zvecr) [inline keyboards/skmt/15k/skmt_15k_macropad.json:1]: This should be removed as it does not belong in this repo.

File: keyboards/skmt/15k/rules.mk [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# This file intentionally left blank
+
+# RGBLIGHT_ENABLE = yes
```
Comment (zvecr) [inline keyboards/skmt/15k/rules.mk:3]:
```suggestion
```

File: keyboards/skmt/15k/keymaps/qwert/keymap.c [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    [2] = LAYOUT_default(
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    ),
+    [3] = LAYOUT_default(
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    )
```
Comment (zvecr) [inline keyboards/skmt/15k/keymaps/qwert/keymap.c:34]:
```suggestion
    )
```

File: keyboards/skmt/15k/keymaps/fns/keymap.c [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    [2] = LAYOUT_default(
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    ),
+    [3] = LAYOUT_default(
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    )
```
Comment (zvecr) [inline keyboards/skmt/15k/keymaps/fns/keymap.c:34]:
```suggestion
    )
```

File: keyboards/skmt/15k/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    [2] = LAYOUT_default(
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    ),
+    [3] = LAYOUT_default(
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+        KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    )
```
Comment (zvecr) [inline keyboards/skmt/15k/keymaps/default/keymap.c:35]:
```suggestion
    )
```

File: keyboards/skmt/15k/15k.c [hunk 7]
```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Jimmy Lye (@satorusaka)
```
Comment (zvecr) [inline keyboards/skmt/15k/15k.c:1]: This file can be removed

File: keyboards/skmt/15k/info.json [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+                { "matrix": [1, 3], "x": 3, "y": 1 },
+                { "matrix": [1, 4], "x": 4, "y": 1 },
+                { "matrix": [2, 0], "x": 0, "y": 2 },
+                { "matrix": [2, 1], "x": 1, "y": 2 },
+                { "matrix": [2, 2], "x": 2, "y": 2 },
+                { "matrix": [2, 3], "x": 3, "y": 2 },
+                { "matrix": [2, 4], "x": 4, "y": 2 }
+            ]
+        }
+    }
```
Comment (zvecr) [inline keyboards/skmt/15k/info.json:46]:
```suggestion
    },
    "rgb_matrix": {
        "driver": "WS2812",
        "layout": [
            { "flags": 4, "matrix": [2, 4], "x": 224, "y": 64 },
            { "flags": 4, "matrix": [2, 3], "x": 168, "y": 64 },
            { "flags": 4, "matrix": [2, 2], "x": 112, "y": 64 },
            { "flags": 4, "matrix": [2, 1], "x": 56, "y": 64 },
            { "flags": 4, "matrix": [2, 0], "x": 0, "y": 64 },
            { "flags": 4, "matrix": [1, 4], "x": 224, "y": 32 },
            { "flags": 4, "matrix": [1, 3], "x": 168, "y": 32 },
            { "flags": 4, "matrix": [1, 2], "x": 112, "y": 32 },
            { "flags": 4, "matrix": [1, 1], "x": 56, "y": 32 },
            { "flags": 4, "matrix": [1, 0], "x": 0, "y": 32 },
            { "flags": 4, "matrix": [0, 4], "x": 224, "y": 0 },
            { "flags": 4, "matrix": [0, 3], "x": 168, "y": 0 },
            { "flags": 4, "matrix": [0, 2], "x": 112, "y": 0 },
            { "flags": 4, "matrix": [0, 1], "x": 56, "y": 0 },
            { "flags": 4, "matrix": [0, 0], "x": 0, "y": 0 }
        ]
    }
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/skmt/15k/rules.mk [hunk 9]
```diff
@@ -0,0 +1,4 @@
+RGB_MATRIX_ENABLE = yes
```
Comment (zvecr) [inline keyboards/skmt/15k/rules.mk:1]:
```suggestion
```

File: keyboards/skmt/15k/readme.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+* Hardware Supported: skmt_15k_macropad on rpi pico
+* Hardware Availability: [satorusaka/15k-macropad](https://github.com/satorusaka/15k-macropad)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make skmt_15k_macropad:default
+
+Flashing example for this keyboard:
+
+    make skmt_15k_macropad:default:flash
```
Comment (zvecr) [inline keyboards/skmt/15k/readme.md:18]:
```suggestion
    make skmt/15k:default

Flashing example for this keyboard:

    make skmt/15k:default:flash
```

File: keyboards/skmt/15k/info.json [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+    "maintainer": "satorusaka",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
```
Comment (zvecr) [inline keyboards/skmt/15k/info.json:13]:
```suggestion
        "nkro": true,
        "rgb_matrix": true
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**